### PR TITLE
Redesign: add feedback dialog & button in tag panel

### PR DESCRIPTION
**Pull request by bwindels**
https://github.com/vector-im/riot-web/issues/7896

### CODE REVIEW

**Comment by bwindels** [comment]
build seems broken to changing electron dependencies on develop, odd...

File: res/css/structures/_TagPanel.scss [hunk 1]
```diff
@@ -124,10 +124,24 @@ limitations under the License.
     padding-right: 4px;
 }
 
+.mx_TagPanel_groupsButton {
+    flex: 0;
+    margin: 17px 0 3px 0;
+}
+
```
Comment (jryans) [inline res/css/structures/_TagPanel.scss]: Nit: Maybe remove extra blank line?

File: src/components/views/dialogs/RedesignFeedbackDialog.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import QuestionDialog from './QuestionDialog';
+import { _t } from '../../../languageHandler';
+
+export default (props) => {
+
```
Comment (jryans) [inline src/components/views/dialogs/RedesignFeedbackDialog.js]: Nit: Blank line here, not sure if intended...

File: src/components/views/dialogs/RedesignFeedbackDialog.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    const existingIssuesUrl = "https://github.com/vector-im/riot-web/issues" +
+        "?q=is%3Aopen+is%3Aissue+label%3Aredesign+sort%3Areactions-%2B1-desc";
+    const newIssueUrl = "https://github.com/vector-im/riot-web/issues/new" +
+        "?assignees=&labels=redesign&template=redesign_issue.md&title=";
+
+    const description1 =
+        _t("Thanks for testing the Riot Redesign. " +
```
Comment (jryans) [inline src/components/views/dialogs/RedesignFeedbackDialog.js]: Are we intending to call it "Riot Redesign" like a product name, or would "Riot redesign" be better?

Comment (bwindels) [reply src/components/views/dialogs/RedesignFeedbackDialog.js]: I think the copy might be adjusted later on, for now I just took the wording from [the design](https://github.com/vector-im/riot-web/issues/7896).

**Pull request merged**